### ISSUE TITLE
Show tables records top 100 and present them on page

### ISSUE THREAD

**Issue by rafalkasa** [schemaspy/schemaspy#17]

**Comment by cheoAlejo** [comment]
+1
Well, it could be optional, like passing an argument specifying the number of rows to show.

Example showing first 10 rows:
```
java -jar schemaspy.jar -rows 10
```

**Comment by danwatt** [comment]
It looks like someone submitted a patch to add this support to Schema Spy 5:

https://sourceforge.net/p/schemaspy/feature-requests/85/

**Comment by anothermindbomb** [comment]
This should be turned off by default and only enabled via a switch, and I'd argue that it shouldn't be added at all.

SchemaSpy is a product for creating entity relationship diagrams, and (possibly live) table contents have no place on an ERD.

If this is enabled, then the end user runs the risk of exposing production data to anyone who is able to view the ERD. For example, I don't want my business analysts to be able to see any live or production data anywhere other than within the database where their access can be controlled and limited.

**Comment by npetzall** [comment]
I'm with other ones, it's default off since database might contain sensitive data.

**Comment by danwatt** [comment]
This is probably a completely different request, but what if instead there was the ability to do some analysis on the values without revealing the data itself? Ex: how many nulls vs non-nulls are present in a column? What is the longest, shortest, and average length of a varchar, etc?

Something like what MYSQL does with `... PROCEDURE ANALYZE` (or if you have ever used PhpMyAdmin, the `Propose table structure` button) https://dev.mysql.com/doc/refman/5.6/en/procedure-analyse.html

**Comment by lukeIam** [comment]
> +1
> Well, it could be optional, like passing an argument specifying the number of rows to show.

any update on this?

> https://sourceforge.net/p/schemaspy/feature-requests/85/

seems to be a little bit outdated...

I had a short look to your code but it looks to complicated to implement the push of information through all the classes and templates for me (discovered schemaspy today).

**Comment by npetzall** [comment]
@lukeIam the model and everything was never intended to include data from the database. I commented that it should be off, but if I'm honest I really don't want to include it. Since data and structures are different things.

**Comment by lukeIam** [comment]
@npetzall fair enough - I'm currently using schemaspy for exploring an unknown database and the column titles aren't the best, so I have to take a look at the data too.

**Comment by npetzall** [comment]
@lukeIam have a look at https://dbeaver.io/

**Comment by alanreyesv** [comment]
> 
> 
> @lukeIam the model and everything was never intended to include data from the database. I commented that it should be off, but if I'm honest I really don't want to include it. Since data and structures are different things.

It can be useful when using development database to generate the data dictionary to display example data.

Use cases are:
- The type is string, but format is an ULID (`01D78XYFJ1PRM1WPBCBT3VHMNV`)
- The type is a JSON, with a specific structure.
- The type follows a sequence (incremental versions on a specific element).
